server/schema/customer/res.js

```diff
@@ -1,9 +1,26 @@
 import {Customer} from '../../models'
 import GraphQLDate from 'graphql-date'
 import bcrypt from 'bcrypt'
+import jwt from 'jsonwebtoken'
 import {_auth} from '../../util'
-import {generateLoginJwt} from './mutations'
 const SALT = 10
+async function generateLoginJwt(input, msg = '') {
```

Comment:
input -> customer , best send in "username" only
** and why do we need to pass-in object, while we only need username for searching ?
for the naming scheme "input" is just too unambiguous!